gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/audit/impl/AuditReporterManagerImpl.java

```diff
@@ -242,8 +245,8 @@ private void updateReporterProvider(Reporter reporter, GraviteeContext context)
     public io.gravitee.am.reporter.api.provider.Reporter getReporter() {
        return reporters.entrySet()
                 .stream()
-                .map(entry -> entry.getValue())
-                .filter(reporter -> reporter.canSearch())
+                .map(Entry::getValue)
+                .filter(io.gravitee.am.reporter.api.provider.Reporter::canSearch)
```

Comment:
The lambda was less verbose for this one 🤷

Reply:
I think I need to rename this class to be ISO with other plugins
```suggestion
                .filter(ReporterProvider::canSearch)
```